centrallix/tests/test_moneytype_01.c

```diff
@@ -0,0 +1,44 @@
+#include "obj.h"
+#include <assert.h>
+#include <stdio.h>
+//#include "xstring.h"
+//#include "mtsession.h"
+//#include "mtask.h"
+
+long long
+test(char** name)
+{
+    *name = "moneytype_01 - objDataToInteger";
+    int testValue;
```

Comment:
Could you give this variable a more descriptive name (like `testInt` or something else indicating that it's the integer output), or just remove it (e.g. `assert(objDataToInteger(7,&test,NULL) == 0)`)?